src/modules/routing/RoutingModule.ts

```diff
@@ -49,18 +46,14 @@ export class RoutingModule {
       const { verkey, invitationUrl, alias = 'Mediator' } = mediatorConfiguration
       const mediatorInvitation = await ConnectionInvitationMessage.fromUrl(invitationUrl)
 
-      const connection = await this.connectionService.processInvitation(mediatorInvitation, { alias })
-      const {
-        message: connectionRequest,
-        connectionRecord: connectionRecord,
-      } = await this.connectionService.createRequest(connection.id)
-      const connectionResponse = await this.messageSender.sendAndReceiveMessage(
-        createOutboundMessage(connectionRecord, connectionRequest, connectionRecord.invitation),
-        ConnectionResponseMessage
-      )
-      await this.connectionService.processResponse(connectionResponse)
-      const { message: trustPing } = await this.connectionService.createTrustPing(connectionRecord.id)
-      await this.messageSender.sendMessage(createOutboundMessage(connectionRecord, trustPing))
+      const connectionRecord = await this.connectionService.processInvitation(mediatorInvitation, { alias })
+      const { message: connectionRequest } = await this.connectionService.createRequest(connectionRecord.id)
+
+      const outboundMessage = createOutboundMessage(connectionRecord, connectionRequest, connectionRecord.invitation)
+      outboundMessage.payload.setReturnRouting(ReturnRouteTypes.all)
```

Comment:
You're already setting return routing if the connection has no inbound endpoint. this shouldn't be needed right?

Reply:
I had to move it into `Dispatcher`, where is not called with this manual invitation acceptance process, because when `return_route` is set, we don't call `sendMessage` where I had it previously in #218